mubench.reviewsite/tests/StoreReviewTest.php

```diff
@@ -23,7 +23,7 @@ class StoreReviewTest extends DatabaseTestCase
             0 => [
                 'hit' => 'Yes',
                 'types' => [
-                    'missing/call'
+                    '1'
```

Comment:
missing/call is the first to get inserted in our init_db.sql so it should get the id 1